### ISSUE TITLE
Feed not showing Received polls

### ISSUE THREAD

**Issue by rickardinho** [DRDD2016/native#195]
Joined an event using Code, then voted. Got “Thanks for voting message” and then nothing for this event is appearing in the feed.  Looks like the Feed is not showing "Received" polls, but only Hosting.

**Comment by rickardinho** [comment]
Feed and Calendar both look like they aren't showing any received polls, only hosting ones.

**Comment by minaorangina** [comment]
@rickardinho We have mimicked the same functionality that was in the original web app.  If you refer back to it, you'll see the calendar only displays confirmed, finalised events.  It's not possible to chronologise polls because they might have several date options.

The web app feed tells you the following:
- you have created a poll
- you have created a finalised event
- you have edited a finalised event

- someone has voted on your poll
- someone has responded (rsvp'd) to your finalised event
- someone wants you to vote on their poll
- someone has invited you to their finalised event
- someone has edited their event

There is no functionality for the feed informing you that you've voted on someone else's poll.  With this being the case, I'll close this issue.

**Comment by rickardinho** [comment]
Yes, you're right. However, should I at least see  "someone wants you to vote on their poll" in the feed after entering the code?

**Comment by minaorangina** [comment]
@rickardinho this would be a new feature if you wanted it, as in the original app that message is delivered after a different set of circumstances (namely inviting a fb friend).  Would need additional server work.

Though from a user experience point of view, it could be confusing to receive "X wants you to vote on their poll" after you've already entered the code and voted.  Would make the app seem laggy/out-of-sync.

This might well be something worth investigating during testing - see what extra messages, if any, are needed to guide the user experience, then go from the feedback?

**Comment by rickardinho** [comment]
I see, i think with the new flow this message should be, "you voted on X's poll" or you replied to X's message, otherwise the event is lost from view completely and there is no way of knowing if you voted or not... without voting again, which is very clunky.  There may be other messages too as you say, and I might revisit when determining which messages should be push notifications and which are better left in the feed - can I leave this open for later reference... I'll remove the bug label though.

**Comment by minaorangina** [comment]
👍
